src/main/java/com/microsoft/aad/msal4j/ConfidentialClientApplication.java

```diff
@@ -39,6 +39,10 @@
 
 import static com.microsoft.aad.msal4j.ParameterValidationUtils.validateNotNull;
 
+/**
+ * Class to be used to acquire tokens for confidential client applications (Web Apps, Web APIs,
+ * and daemon applications).
```

Comment:
should have an aka.ms link that points to the wiki or public docs describing the confidential client

Reply:
Added links to the Java wiki in both PCA and CCA files.